webpack.config.babel.js

```diff
@@ -42,6 +42,7 @@ const basePlugins = [
 if (ENV === 'production') {
   basePlugins.push(
     new webpack.optimize.UglifyJsPlugin({
+      sourceMap: true,
```

Comment:
makes sure the minifying plugin does not mess with the source maps. But I still haven't completely solved the issue, I'm still not getting perfect source maps on Chrome.